telegram/ext/commandhandler.py

```diff
@@ -124,14 +128,19 @@ def __init__(self,
                  pass_update_queue=False,
                  pass_job_queue=False,
                  pass_user_data=False,
-                 pass_chat_data=False):
+                 pass_chat_data=False,
+                 description=None):
         super(CommandHandler, self).__init__(
             callback,
             pass_update_queue=pass_update_queue,
             pass_job_queue=pass_job_queue,
             pass_user_data=pass_user_data,
             pass_chat_data=pass_chat_data)
 
+        self.description = description
+        if self.description is not None and not 3 <= len(self.description) <= 256:
+            raise ValueError('Command description is not valid.')
```

Comment:
I should remove this. We don't usually perform such checks as TG will just return BadRequest if invalid and more importantly, the limitations might change